.github/workflows/integration.yaml

```diff
@@ -1,26 +1,28 @@
-name: Actions-CI 
+name: Actions-CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master, v5]
   pull_request:
-    branches: [ master ]
+    branches: [master, v5]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [latest, lts/*, lts/-1]
```

Comment:
This is fancy, but we probably want to hard-code the versions we're running against. Given the history of the project, it feels unlikely that we're going to keep up!

(Approving anyway since I'll be gone.)